src/instructlab/configuration.py

```diff
@@ -127,6 +127,58 @@ def after_debug_level(self):
         return self
 
 
+class _chat_rag_retriever_embedder(BaseModel):
```

Comment:
this one could be reused for ingestion purposes: can we rename it and remove the `chat_` part?